src/example-types/metricGroups/pivot.js

```diff
@@ -297,16 +273,6 @@ let createPivotScope = (node, schema, getStats) => {
       async (children, group, index, groups) => {
         // Defaulting the group size to be 10
         if (!group.size) group.size = 10
```

Comment:
What is this about? Not every type has a size prop. Setting default prop values is a per type concern and can happen during destructing in type specific implementations

Reply:
This code was here before this PR, I'm not sure what the context is for the sizing.